docs/release_notes/v0.10.0.md

```diff
@@ -0,0 +1,471 @@
+
+To see the full documentation regarding upgrading Dapr, see [here](https://github.com/dapr/docs/tree/master/howto/deploy-k8s-prod#upgrading-dapr-with-helm).
+
+##### 1. Get the latest CLI
+
+Get the latest version of the Dapr CLI as outlined above, and put it in your path.
+You can also use the helper scripts outlined [here](https://github.com/dapr/cli#installing-dapr-cli) to get the latest version.
+
+##### 2. Save existing certificates
+
+Using the Dapr CLI, export your certificates:
+
+```
+dapr mtls export -o ./certs
+```
+
+##### 3. Upgrade
+
+First, update your Helm repos:
+
+```
+helm repo update
+```
+
+Run the following command to upgrade Dapr:
+
+```
+cd certs
+
+helm upgrade dapr dapr/dapr --version 0.10.0 --namespace dapr-system --reset-values --set-file dapr_sentry.tls.root.certPEM=ca.crt --set-file dapr_sentry.tls.issuer.certPEM=issuer.crt --set-file dapr_sentry.tls.issuer.keyPEM=issuer.key
+```
+
+Wait until all the pods are in Running state:
+
+```
+kubectl get pods -w -n dapr-system
+```
+
+Verify the contorl plane is updated and healthy:
+
+```
+dapr status -k
+```
+
+Next, issue a rolling update to your Dapr enabled deployments. When the pods restart, the new sidecar version will be picked up.
+
+```
+kubectl rollout restart deploy/<DEPLOYMENT-NAME>
+```
+
+All done!
+
+#### Starting fresh
+
+If you previously installed Dapr on your Kubernetes cluster using the Dapr CLI, run:
+
+*Note: Make sure you're uninstalling with your existing CLI*
+
+```bash
+dapr uninstall --kubernetes
+```
+
+It's fine to ignore any errors that might show up.
+
+If you previously installed Dapr using __Helm 2.X__:
+
+```bash
+helm del --purge dapr
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+Update the Dapr repo:
+
+```bash
+helm repo update
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, you can use the CLI:
+
+```
+dapr init --kubernetes
+```
+
+#### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+dapr status -k
+
+dapr-sentry            dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21  
+dapr-operator          dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21  
+dapr-sidecar-injector  dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21  
+dapr-placement         dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21 
+```
+
+After Dapr 0.10.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
```

Comment:
Do we want to call out the flag changes in daprd / CLI ?

Reply:
There's no doc that captures that we can send people to, as it's basically all over.

Reply:
But that's not a breaking change until v1.0, we still support the old flags

Reply:
That's also true.

Reply:
ok, great